beaver_store/lib/src/storage_service/gcloud_storage_service.dart

```diff
@@ -71,24 +73,54 @@ class BeaverBuild extends Model {
 class GCloudStorageService implements StorageService {
   DatastoreDB _db;
 
+  Future<String> _allocateProjectId() async {
```

Comment:
If that format is okay, then I want to try to use, instead of String, datastore.Key which is unique for an entity, has some benefits for the platform.